src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Overview/Cells/DiaryOverviewDayTableViewCell.swift

```diff
@@ -8,8 +8,18 @@ class DiaryOverviewDayTableViewCell: UITableViewCell {
 
 	// MARK: - Internal
 
-	func configure(cellViewModel: DiaryOverviewDayCellModel) {
+	func configure(
+		cellViewModel: DiaryOverviewDayCellModel,
+		didTapClickableView: @escaping () -> Void
+	) {
+		self.didTapClickableView = didTapClickableView
+
 		dateLabel.text = cellViewModel.formattedDate
+		dateLabel.accessibilityIdentifier = String(format: AccessibilityIdentifiers.ContactDiaryInformation.Overview.cellDateHeader, cellViewModel.accessibilityIdentifierIndex)
+		let tapOnDateStackViewRecognizer = UITapGestureRecognizer(target: self, action: #selector(clickableAreaWasTapped))
+		dateStackView.addGestureRecognizer(tapOnDateStackViewRecognizer)
+		dateStackView.addSubview(dateLabel)
```

Comment:
Isn't the `dateLabel` already added from InterfaceBuilder?

Reply:
You are right. I removed it.